jobs/config.json

```diff
@@ -2763,6 +2763,12 @@
     ], 
     "scenario": "kubernetes_e2e"
   }, 
+  "ci-perf-tests-kubemark-100-benchmark": {
+    "args": [
+      "./benchmark/runner.sh"
```

Comment:
The runner.sh file doesn't use docker to run the binary. It runs the binary directly (the only docker it uses is for containerized compiling, but that is different).
Ref: https://github.com/shyamjvs/perf-tests/blob/48ba2fda680939e11b607daeb7dad16297f9749e/benchmark/runner.sh